paasta_tools/generate_deployments_for_service.py

```diff
@@ -184,6 +184,8 @@ def get_desired_state(service, branch, remote_refs, deploy_group):
     # paasta-paasta-cluster.instance
```

Comment:
It looks like we have 0 coverage on this (important) function. Can you write at least one test for this?
```
def test_get_disired_state_understands_tags():
   remote_refs = (get some legit remote refs with all sorts of crazy stuff)
  service = fake_service
  branch = ?
  deploy_group = ?
  expected_desired_state = 'stopped'
  actual = ....


def get_desired_state_fails_gracefully_with_start():
  ....
  actual = ...
  assert actual == ("start", None)

```